include/NovelRT/NovelRunner.h

```diff
@@ -48,6 +48,7 @@ namespace NovelRT
          */
         explicit NovelRunner(int32_t displayNumber,
                              const std::string& windowTitle = "NovelRTTest",
+                             NovelRT::Windowing::WindowMode windowMode = NovelRT::Windowing::WindowMode::Windowed,
```

Comment:
You're inside the root namespace here, you should be able to drop the root namespace from the window mode declaration here?